.Rbuildignore

```diff
@@ -20,3 +20,4 @@
 ^docs$
 ^CRAN-RELEASE$
 ^cran-comments\.md$
+^raw-data$
```

Comment:
can this be done with `data-raw`? Thank you

Reply:
What's `data-raw`?

Reply:
ggplot2 uses the folder called `data-raw` (vs `raw-data`) 

https://github.com/tidyverse/ggplot2/tree/master/data-raw